src/renderer/components/ConfigureBeaconNode/InputBeaconNode.tsx

```diff
@@ -31,7 +32,12 @@ export const InputBeaconNode: React.FunctionComponent<IInputBeaconNodeProps> = (
         }
 
         try {
-            const network = await readBeaconChainNetwork(beaconNodeInput);
+            // eslint-disable-next-line camelcase
+            const response = await axios.get<{data: {genesis_fork_version: string}}>(`/eth/v1/beacon/genesis`, {
```

Comment:
we should at least move this to some static method on cgeth2client?

Reply:
Yeah but it would be quite difficult to find this line of code in case response or url changes